src/picongpu/include/traits/attribute/GetCharge.hpp

```diff
@@ -100,8 +100,9 @@ HDINLINE float_X getCharge(const float_X weighting, const T_Particle& particle)
     typedef T_Particle ParticleType;
     typedef typename PMacc::traits::HasIdentifier<ParticleType, boundElectrons>::type hasBoundElectrons;
```

Comment:
In `speciesDefinition.param` we might add another comment for the `chargeRatio`. It should probably state that charge calculation is not affected by this value if `boundElectrons` is set as a species attribute.